app/react/package.json

```diff
@@ -19,12 +19,14 @@
   },
   "scripts": {
     "dev": "DEV_BUILD=1 nodemon --watch ./src --exec 'npm run prepublish'",
-    "prepublish": "node ../../scripts/prepublish.js"
+    "prepublish": "node ../../scripts/prepublish.js",
+    "test": "../../node_modules/.bin/jest --watch"
   },
   "dependencies": {
     "@storybook/addon-actions": "^3.1.6",
     "@storybook/addon-links": "^3.1.6",
     "@storybook/addons": "^3.1.6",
+    "@storybook/addon-options": "^3.1.6",
```

Comment:
@igor-dv, it's a very good question!

addon-options is a very special package - it doesn't provide any additional functionality like `actions` and `lilnks`. We use it to get access to UI API. 

So maybe better to think that it's not an addon at all!? :thinking: 

For Storybook users it's just one kind of API. Common use:
```js
import { configure, storiesOf } from '@storybook/react';
import { setOptions } from '@storybook/addon-options'; // why I need one more package?
```

Nevertheless, the only my reason here is to provide such API:
```js
setOptions({
  hierarchySeparator: /\/|:\//, // regular expression to separate stories nesting
  multistorySeparator: /:/, // regular expression to separate multistories section
  previewDecorator: stories => <div>{stories}</div>, // root decorator for preview
});
```

I guess it's much more comfortable for consumers to have all similar APIs in one place, than for example:

```js
import { setOptions } from '@storybook/addon-options';
import { setPreferences } from '@storybook/react';

setOptions({
  hierarchySeparator: /\/|:\//, // regular expression to separate stories nesting
});

setPreferences({
  multistorySeparator: /:/, // regular expression to separate multistories section
  previewDecorator: stories => <div>{stories}</div>, // root decorator for preview
});

```

The last example is more consistent with the project design, but it can :suspect: >> :boom: >> :feelsgood: